Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -762,6 +764,14 @@ def expanded(feat):
         indices = inst_by_id[inst_id]
         instance = newtable[i]
 
+        for var in domain.variables + domain.metas:
+            if var in idvarlist:
+                continue
+            if numpy.isnan(instance[var]):
+                for ind in indices:
+                    if not numpy.isnan(table[ind, var]):
```

Comment:
I'm getting an error on this line when changing from Use identifiers to Use instance equality.
ValueError: 'name (Naive Bayes)' is not in domain

Reproduce the error: Three datasets on the input of Venn diagram. Select "Use identifiers". Select a subset that is intersection of only two datasets. Select "Use instance equality". Change selected subset.